plugin/trino-resource-group-managers/src/main/java/io/trino/plugin/resourcegroups/db/ResourceGroupsDao.java

```diff
@@ -106,4 +57,115 @@ String getExactMatchResourceGroup(
             @Bind("environment") String environment,
             @Bind("source") String source,
             @Bind("query_type") String queryType);
+
+    @SqlUpdate("INSERT INTO resource_groups_global_properties\n" +
```

Comment:
Why all these changes? It seems unrelated to Flyway. At a minimum, please move to a separate commit that explains the motivation.

Reply:
I made these changes so I could get tests running using the MySQL DAO instead of the H2 one. I backed this change out though for now.